packages/table/src/cell/_common.scss

```diff
@@ -18,12 +18,15 @@ $dark-cell-background-color: $dark-gray3 !default;
 $dark-cell-border-color: $pt-dark-divider-black !default;
 $dark-cell-text-color: $pt-dark-text-color !default;
 
+$text-fade-in-transition: color $pt-transition-duration * 3;
```

Comment:
hm in that case i'd like to use this variable inside `_loading.scss` as well, though it feels weird to have that file import from this one